logOutputSubject.go

```diff
@@ -4,10 +4,16 @@ import (
 	"fmt"
 	"io"
 	"sync"
+	"unicode/utf8"
 
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 )
 
+const ASCIISpace = 32
```

Comment:
This could also be changed to keep consistency with others:

```suggestion
const ASCIISpace = byte(' ')
```

Reply:
done